base/linalg/dense.jl

```diff
@@ -603,16 +603,16 @@ systems. For example: `A=factorize(A); x=A\\b; y=A\\C`.
 
 | Properties of `A`          | type of factorization                          |
 |:---------------------------|:-----------------------------------------------|
-| Positive-definite          | Cholesky (see [`cholfact`](:func:`cholfact`))  |
-| Dense Symmetric/Hermitian  | Bunch-Kaufman (see [`bkfact`](:func:`bkfact`)) |
-| Sparse Symmetric/Hermitian | LDLt (see [`ldltfact`](:func:`ldltfact`))      |
+| Positive-definite          | Cholesky (see [`cholfact`](@ref))  |
```

Comment:
The markdown parser is robust enough to handle this kind of thing, but can be re-aligned during one of the PRs I've got lined up to polish everything up that isn't *quite* right. I figured that was probably the simplest way handle details such as this one.